### PR TITLE
Introduce mobile web clips

### PR DESCRIPTION
**Pull request by JustynaBroniszewska**

### CODE REVIEW

File: universal-login-wallet/src/config/config.ts [hunk 1]
```diff
@@ -5,8 +5,8 @@ export default Object.freeze({
 
   development: {
     domains: ['mylogin.eth'],
-    relayerUrl: 'http://localhost:3311',
-    jsonRpcUrl: 'http://localhost:18545',
+    relayerUrl: 'http://192.168.1.23:3311',
+    jsonRpcUrl: 'http://192.168.1.23:18545',
```
Comment (rzadp) [inline universal-login-wallet/src/config/config.ts]: Remove

File: universal-login-wallet/package.json [hunk 2]
```diff
@@ -6,7 +6,7 @@
   "scripts": {
     "start": "webpack-dev-server --mode development",
     "prebuild": "yarn clean",
-    "build": "webpack --mode production --progress && cp \"src/_redirects\" dist/html/_redirects",
+    "build": "webpack --mode production --progress ",
```
Comment (xitronix) [inline universal-login-wallet/package.json]: Extra whitespace

File: universal-login-wallet/webpack.config.js [hunk 3]
```diff
@@ -3,6 +3,7 @@ const MiniCssExtractPlugin = require("mini-css-extract-plugin")
 const Dotenv = require("dotenv-webpack")
 const webpack = require('webpack');
 const path = require('path');
+const CopyPlugin = require('copy-webpack-plugin');
```
Comment (rzadp) [inline universal-login-wallet/webpack.config.js]:
```suggestion
const copyPlugin = require('copy-webpack-plugin');
```

File: universal-login-wallet/webpack.config.js [hunk 4]
```diff
@@ -61,7 +62,11 @@ module.exports = {
       'process.env.RELAYER_URL': JSON.stringify(process.env.RELAYER_URL),
       'process.env.ENS_DOMAIN_1': JSON.stringify(process.env.ENS_DOMAIN_1),
       'process.env.JSON_RPC_URL': JSON.stringify(process.env.JSON_RPC_URL)
-    })
+    }),
+    new CopyPlugin([
```
Comment (rzadp) [inline universal-login-wallet/webpack.config.js]:
```suggestion
    new copyPlugin([
```

File: universal-login-wallet/src/config/config.ts [hunk 5]
```diff
@@ -5,8 +5,8 @@ export default Object.freeze({
 
   development: {
     domains: ['mylogin.eth'],
-    relayerUrl: 'http://localhost:3311',
-    jsonRpcUrl: 'http://localhost:18545',
+    relayerUrl: 'http://192.168.1.23:3311',
```
Comment (xitronix) [inline universal-login-wallet/src/config/config.ts]: Why did we change it?

File: universal-login-wallet/webpack.config.js [hunk 6]
```diff
@@ -61,7 +62,11 @@ module.exports = {
...
+      {from: './src/ui/assets/jarvis-logo.png'},
+      {from: './src/_redirects'}
```
Comment (rzadp) [inline universal-login-wallet/webpack.config.js]: Are they copied in the same directory as before, so `dist/html/` ?

Comment (JustynaBroniszewska) [reply universal-login-wallet/webpack.config.js]: Yes

File: universal-login-wallet/src/config/config.ts (hunk 5 above)
Comment (JustynaBroniszewska) [reply universal-login-wallet/src/config/config.ts]: My bad. It shouldn't be here.

File: universal-login-wallet/webpack.config.js (hunk 3 above)
Comment (JustynaBroniszewska) [reply universal-login-wallet/webpack.config.js]: This is constructor

**Pull request merged**